### PR TITLE
[WIP]feat: change deploy mode from branch to tag

### PR DESCRIPTION
**Pull request by claytonsilva**
we change deploy from branch to tag, this feature will secure our
deploys using tag, not branch more.

### CODE REVIEW

File: scm/git/git.go [hunk 1]
```diff
@@ -7,11 +7,10 @@ import (
 	"regexp"
 
 	"github.com/mitchellh/mapstructure"
-	"gopkg.in/src-d/go-git.v4"
-	"gopkg.in/src-d/go-git.v4/plumbing"
-
 	"github.com/pagarme/deployer/pipeline"
 	"github.com/pagarme/deployer/scm"
+	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/plumbing"
```
Comment (derekstavis) [inline scm/git/git.go]: Unrelated change

Comment (claytonsilva) [reply scm/git/git.go]: no done yet, change do WIP
